x-pack/plugins/enterprise_search/server/routes/enterprise_search/indices.ts

```diff
@@ -515,7 +516,9 @@ export function registerIndexRoutes({
         pipeline: { description: defaultDescription, ...pipeline },
       };
 
-      const simulateResult = await client.asCurrentUser.ingest.simulate(simulateRequest);
+      const simulateResult: IngestSimulateResponse = await client.asCurrentUser.ingest.simulate(
```

Comment:
question: do you actually need to explicitly type this? Shouldn't this just be inferred from the call itself?

Reply:
That may just be a caching/slow typing thing, but if the client isn't typed we have a bigger problem 👀

Reply:
confirmed, slow typing thing. I will double check next time.